logic/service/data-service.js

```diff
@@ -188,11 +188,15 @@ exports.DataService = Montage.specialize(/** @lends DataService.prototype */{
         value: function () {
             var children = this.rootService._childServices.get(DataObjectDescriptor.ALL_TYPES),
                 offline, i, n;
-            for (i = 0, n = children.length; i < n && !offline; ++i) {
-                if (children[i].worksOffline) {
-                    offline = children[i];
+
```

Comment:
No blank lines within methods. If your method requires a blank line, it's doing too many things and its functionality should be broken out into separate methods.